docs/zh/UserGuide/QuickStart/Command-Line-Interface.md

```diff
@@ -159,7 +181,7 @@ curl -X POST "http://{your-keycloack-server}/auth/realms/{your-realm}/protocol/o
  -d "username={username}" \
  -d "password={password}" \
  -d 'grant_type=password' \
- -d "client_id=iotdb"
+ -d "client_id=iotdb-client"
```

Comment:
Should we alter this line in english doc?

Reply:
Sorry, I forgot to modify it. I'll modify it